itest/lnd_open_channel_test.go

```diff
@@ -877,6 +877,114 @@ func testFundingExpiryBlocksOnPending(ht *lntest.HarnessTest) {
 	ht.MineBlocksAndAssertNumTxes(1, 1)
 }
 
+// testConfirmationsUntilActiveOnPending verifies that as a channel status
+// transitions from pending to active, the ConfirmationsUntilActive field of the
+// PendingChannel decreases to 0 as expected.
+func testConfirmationsUntilActiveOnPending(ht *lntest.HarnessTest) {
+	var (
+		numConfs uint32         = 5
+		chanAmt  btcutil.Amount = 100000
+	)
+
+	// As we need to create a channel that requires more than 1 confirmation
+	// before it's open, with the current set of defaults, we'll need to
+	// create a new node instance.
+	alice := ht.NewNodeWithCoins("Alice", nil)
+	bob := ht.NewNode("Bob", []string{
```

Comment:
I don't think you need to start a new node, but you can change it also in the OpenChannel parameters ?

Reply:
ok however I think the comment is a bit misleading, because there are no default nodes anymore. We need to create nodes now.